management-api/src/main/java/com/afh/skytouch/managment/controllers/GenericProductController.java

```diff
@@ -0,0 +1,33 @@
+package com.afh.skytouch.managment.controllers;
+
+import com.afh.skytouch.commons.dto.GenericProduct;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.UUID;
+
+@Controller
+@RequestMapping("/genericProduct")
+public class GenericProductController {
+
+    @GetMapping("/{uuid}")
+    public ResponseEntity<?> getProduct(@PathVariable UUID uuid){
+        return ResponseEntity.ok().build();
+    }
+
+    @PostMapping("")
```

Comment:
I think, you should add a verb here to denote the action you are performing, that would also make the code more readable and expressive

Reply:
I tried to follow the convention for REST API https://medium.com/hashmapinc/rest-good-practices-for-api-design-881439796dc9, but I already changed it and added the verb "add". what would be the best practice in an mvc app?